plugins/modules/ec2_vpc_route_table.py

```diff
@@ -665,7 +693,7 @@ def ensure_route_table_present(connection, module):
         changed = True
         if not module.check_mode:
             try:
```

Comment:
Catch WaiterError?

I have one question to ask if you could answer: is there any motivation behind setting all the parameters to `None` (for example `ensure_routes()`)? The case when one parameter is `None` I don't think is handled.

Reply:
I'll get the extra catch added.

Not sure what you're referring to about the None parameters.  I didn't change most of that just to minimise the chances of accidental breakage